docs/docs/policies/actions.md

```diff
@@ -0,0 +1,40 @@
+# Actions
+
+Policies support global actions and specific actions per event. The supported actions now are:
```

Comment:
Could we give a definition of what an action is? I mean people will be able to understand as they start reading the first sentences but I think it is better if we don't let any user guess anything but for us to make the statements